.github/workflows/build-and-test.yml

```diff
@@ -30,7 +26,21 @@ jobs:
 
       - name: Build all on ${{matrix.os}}
         run: make all
-        
+
+
+      - name: Install Snyk CLI
+        run: npm install -g snyk
+
+      - name: Run Snyk security scan
+        run: |
+          snyk auth ${{ secrets.SNYK_TOKEN }}
+          snyk test --all-projects
+
+      - name: Run Snyk Monitor
+        run: |
+          snapshot_link=$(snyk monitor --all-projects | grep "Explore this snapshot at" | sed 's/Explore this snapshot at //')
+          echo "Snyk Monitor link: $snapshot_link" >> $GITHUB_STEP_SUMMARY
```

Comment:
Make this to be markdown syntax for cleaner visible text

Reply:
okayy